src/signals/IncidentMap/components/IncidentMap/IncidentMap.test.tsx

```diff
@@ -30,6 +30,12 @@ describe('IncidentMap', () => {
     expect(screen.getByTestId('incidentMap')).toBeInTheDocument()
   })
 
+  it('renders the GPSLocation', () => {
+    render(withAppContext(<IncidentMap />))
+
+    expect(screen.queryByTestId('gpsButton')).toBeInTheDocument()
```

Comment:
Should be `getByTestId`. You use `query` when asserting a element is not present. 
docs: https://testing-library.com/docs/queries/about/

Reply:
Done, I choose for queryByTestID so you would get a nice return null instead of an error. I thought that would be better, but hé I'm nothing if not flexible. :-)